src/apps/Console/ConsoleFeedback.js

```diff
@@ -243,15 +274,20 @@ export default function ConsoleFeedback({
               Agent #{index + 1}
             </Link>
           ))}
-        <p
-          css={`
-            ${textStyle('body2')}
-            margin: ${GU}px 0 ${GU}px 0;
-          `}
-        >
-          and then pass the parameters required for the agent execute function:
-          the target address, and the human-readable function call.
+      </>
+    )
+  } else if (
+    currentParsedCommand[0] === 'act' &&
```

Comment:
We're doing this check twice right now—could we move the `currentParsedCommand.length` as nested ifs inside the if for `currentParsedCommand[0] === 'act'`?

E.g.

```js
if (currentParsedCommand[0] === 'act') {
  if (currentParsedCommand.length < 2) {
    ...
  } else {
    ...
  }
}
```

Also note the bug in the current implementation—we use both `<= 2` and `>= 2` as conditions, whereas we eliminate this class of bugs from being possible by using the `if/else`.